[2024-06-03T09:51:04.843+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T09:51:04.859+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: f1_data_pipeline_taskflow.load_data__5 manual__2024-06-03T09:50:13.362454+00:00 [queued]>
[2024-06-03T09:51:04.866+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: f1_data_pipeline_taskflow.load_data__5 manual__2024-06-03T09:50:13.362454+00:00 [queued]>
[2024-06-03T09:51:04.866+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T09:51:04.878+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): load_data__5> on 2024-06-03 09:50:13.362454+00:00
[2024-06-03T09:51:04.884+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=528) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T09:51:04.886+0000] {standard_task_runner.py:63} INFO - Started process 565 to run task
[2024-06-03T09:51:04.886+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'f1_data_pipeline_taskflow', 'load_data__5', 'manual__2024-06-03T09:50:13.362454+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpdi0aiz4a']
[2024-06-03T09:51:04.887+0000] {standard_task_runner.py:91} INFO - Job 28: Subtask load_data__5
[2024-06-03T09:51:04.922+0000] {task_command.py:426} INFO - Running <TaskInstance: f1_data_pipeline_taskflow.load_data__5 manual__2024-06-03T09:50:13.362454+00:00 [running]> on host 822b25182b4b
[2024-06-03T09:51:05.029+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='f1_data_pipeline_taskflow' AIRFLOW_CTX_TASK_ID='load_data__5' AIRFLOW_CTX_EXECUTION_DATE='2024-06-03T09:50:13.362454+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-03T09:50:13.362454+00:00'
[2024-06-03T09:51:05.030+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T09:51:05.031+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-06-03T09:51:05.032+0000] {base.py:84} INFO - Using connection ID 'sourcedb_connection' for task execution.
[2024-06-03T09:51:05.078+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T09:51:05.079+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T09:51:05.086+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=f1_data_pipeline_taskflow, task_id=load_data__5, run_id=manual__2024-06-03T09:50:13.362454+00:00, execution_date=20240603T095013, start_date=20240603T095104, end_date=20240603T095105
[2024-06-03T09:51:05.099+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T09:51:05.106+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
